packages/web/src/common/store/player/sagas.ts

```diff
@@ -24,7 +26,13 @@ import {
   delay
 } from 'typed-redux-saga'
 
+import { waitForLibsInit } from 'services/audius-backend/eagerLoadUtils'
+
 import errorSagas from './errorSagas'
+
+// @ts-ignore
+const libs = (): AudiusLibs => window.audiusLibs
```

Comment:
We can't include libs in common sagas direcly afaik, maybe put the necessary functionality in AudiusBackend and use `getContext('audiusBackendInstance)`? cc: @dylanjeffers maybe

Reply:
wonder if we can't put a linter on common to check for `window` usages? Although since we have the `@ts-ignore` we'd probably miss this anyway

Reply:
yeah the move is to grab them from context, happy to show you how, just dm

Reply:
got it! thx. please re-review if latest change makes sense :)